homeworks/nazar_kostiv/HW02/div.py

```diff
@@ -0,0 +1,14 @@
+from pyats.aetest import Testcase, test
+
+
+class Math(Testcase):
+
+    @test
+    def division(self, num1, num2):
+        result = None
```

Comment:
@ProffesorRabbit why did you introduce this variable?

Reply:
@extsoft it's mistake. Removed